mcs/class/System.Net.Http/CFContentStream.cs

```diff
@@ -77,6 +81,7 @@ class CFContentStream : HttpContent
 		public CFContentStream (CFHTTPStream stream)
 		{
 			this.http_stream = stream;
+			this.http_stream.ErrorEvent += HandleErrorEvent;
```

Comment:
Done in the last commit. Of course github does no close the comment 'cause is not that smart.